### PR TITLE
[IOTDB-1059] Support sql statement insert without timestamp

### PR DESCRIPTION
**Pull request by Superainbower**
## Description
1.This PR support users insert data without timestamp, the server use the current system timestamp to pair with the data points.

2.When the timeseries named with time/timestamp/TIME/TIMESTAMP,  it will throw exception.

3.It doesn't support insert mult rows and will throw SQLParserException
 
image

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Comment by qiaojialin** [comment]
Great!
For the second scenario, we could report "Time/Timestamp can not be a series name, please use another name."

**Comment by Superainbower** [comment]
Because IoTDB support create a timeseries named with time/timestamp/TIME/TIMESTAMP now,
So i report "should create it first"  to help me judge "time/timestamp/TIME/TIMESTAMP" is a series name or not

File: server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java [hunk 1]
```diff
@@ -582,6 +582,12 @@ public void recoverFromFailure() {
   @Override
   public void checkIntegrity() throws QueryProcessException {
     super.checkIntegrity();
+    if (this.measurements.length == 1) {
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java:585]: confusing..
how about if the measurements.length >1?
e.g., `insert into (time,  time) values ...  `
how to understand the second `time`?

File: server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java [hunk 2]
```diff
@@ -1763,8 +1763,26 @@ private void parseInsertValuesSpec(InsertValuesSpecContext ctx, InsertOperator i
...
-      } else {
+      } else if (insertMultiValues.get(i).INT() != null) {
         timestamp = Long.parseLong(insertMultiValues.get(i).INT().getText());
+      } else if (insertMultiValues.size() != 1) {
+        throw new SQLParserException("need timestamps when insert multi rows");
+      } else {
+        String timePrecision = IoTDBDescriptor.getInstance().getConfig().getTimestampPrecision();
+        long startupNano = IoTDBDescriptor.getInstance().getConfig().getStartUpNanosecond();
+        switch (timePrecision) {
+          case "ns":
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:1774]: DatetimeUtils.currentTime()

File: server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java (hunk 1 above)
Comment (Superainbower) [reply server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java:585]: emm..
thanks for your advise
it seems need to judge measurements only contain one time column or not

File: server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java [hunk 3]
```diff
@@ -1763,8 +1763,12 @@ private void parseInsertValuesSpec(InsertValuesSpecContext ctx, InsertOperator i
       long timestamp;
       if (insertMultiValues.get(i).dateFormat() != null) {
         timestamp = parseTimeFormat(insertMultiValues.get(i).dateFormat().getText());
-      } else {
+      } else if (insertMultiValues.get(i).INT() != null) {
         timestamp = Long.parseLong(insertMultiValues.get(i).INT().getText());
+      } else if (insertMultiValues.size() != 1) {
+        throw new SQLParserException("need timestamps when insert multi rows");
+      } else {
+        timestamp = parseTimeFormat("now()");
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:1771]:
```suggestion
        timestamp = parseTimeFormat(SQLConstant.NOW_FUNC);
```

Comment (Superainbower) [reply server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:1771]: @HTHou Thank you for your good suggestion

**Review by HTHou — APPROVED** [review]

**Pull request merged**